remote.go

```diff
@@ -864,6 +865,21 @@ func getHavesFromRef(
 	toVisit := maxHavesToVisitPerRef
 	return walker.ForEach(func(c *object.Commit) error {
```

Comment:
Another option would be just to completely ignore any errors returned from this `walker.ForEach` call.